bb-aem-integration-tests/src/test/java/com/cognifide/bdd/demo/aem/touchui/AemTitleTest.java

```diff
@@ -89,11 +90,11 @@ public void tearDown() {
 
   @Test
   public void editTitleComponentTest() {
-    Map<String, List<FieldConfig>> data = page.configureComponent(parsys,
+    ComponentConfiguration data = page.configureComponent(parsys,
         COMPONENT_NAME, COMPONENT_NAME.toLowerCase());
     TitleComponent component =
         (TitleComponent) page.getContent(components.getClazz(COMPONENT_NAME));
     assertThat(component.getTitle(), is(
-        data.get(COMPONENT_NAME).get(0).getValue().toString().toUpperCase()));
+            data.getConfigurationForTab(COMPONENT_NAME).get(0).getValue().toString().toUpperCase()));
```

Comment:
that's because it's the same actually :) I've introduce new constant for that to be more readable